test/nl/surf/eduhub_rio_mapper/smoke_test.clj

```diff
@@ -113,8 +112,13 @@
                   w))
         response))))
 
+(defn- vcr-method [method]
```

Comment:
Deze naam is nogal verwarrend; wat je terugkrijgt is geen vcr method. Noem het dan `make-vcr` of zoiets.